### PR TITLE
Fix processing of sanitizer report files

### PR DESCRIPTION
**Pull request by mpoeter**
### Scope & Purpose

Previously sanitizer reports from TSan/ASan etc. were not processed correctly in the CI. As a result, jobs would fail due to reported warnings, but we were missing the details about what went wrong.

This PR should fix these issues. The reported warnings should now be in the testfailures.txt and also published as CI artifacts.

### CODE REVIEW

File: js/client/modules/@arangodb/testutils/instance-manager.js [hunk 1]
```diff
@@ -334,6 +334,15 @@ class instanceManager {
       this.launchFinalize(startTime);
       return true;
     } catch (e) {
+      // disable watchdog
+      internal.SetGlobalExecutionDeadlineTo(0.0);
```
Comment (dothebart) [inline js/client/modules/@arangodb/testutils/instance-manager.js:338]: should we also set `arangod.serverCrashedLocal`  there? or is this instance only?

File: js/client/modules/@arangodb/testutils/instance-manager.js [hunk 2]
```diff
@@ -334,6 +334,15 @@ class instanceManager {
...
+      this.arangods.forEach(arangod => {
+        try {
+          arangod.terminateInstance();
```
Comment (dothebart) [inline js/client/modules/@arangodb/testutils/instance-manager.js:341]: we should have `true` here for forceterminate?

File: js/client/modules/@arangodb/testutils/instance-manager.js [hunk 3]
```diff
@@ -1001,10 +1010,10 @@ class instanceManager {
           }
           return true;
         }
-        if ((arangod.exitStatus !== null) && (arangod.exitStatus.status === 'RUNNING')) {
-          arangod.exitStatus = statusExternal(arangod.pid, false);
+        if (arangod.isRunning()) {
```
Comment (dothebart) [inline js/client/modules/@arangodb/testutils/instance-manager.js:1013]: can we rather have 
```
arangod.checkIsRunning() /* -> instance.js */ {
if (this.isRunning()) {
  this.exitStatus = this.status(false)
}
```
here?

**Review by dothebart — APPROVED** [review]
I think we should push that functionality to instance.js already. 
else LGTM.

**Review by dothebart — APPROVED** [review]

**Review by KVS85 — APPROVED** [review]

**Pull request merged**